src/engine/channels/enginechannel.cpp

```diff
@@ -23,10 +23,10 @@ EngineChannel::EngineChannel(const ChannelHandleAndGroup& handleGroup,
     m_pMainMix = new ControlPushButton(ConfigKey(getGroup(), "main_mix"));
     m_pMainMix->setButtonMode(ControlPushButton::POWERWINDOW);
     m_pMainMix->addAlias(ConfigKey(getGroup(), QStringLiteral("master")));
-    m_pOrientation = new ControlPushButton(ConfigKey(getGroup(), "orientation"));
+    // crossfader assignment is persistent
+    m_pOrientation = new ControlPushButton(ConfigKey(getGroup(), "orientation"), true);
```

Comment:
```suggestion
    // crossfader assignment is persistent
    m_pOrientation = new ControlPushButton(ConfigKey(getGroup(), "orientation"), true, defaultOrientation);
```

Reply:
:+1: of course, picked the 1st wrong commit and didn't double-check